ebean-core/src/main/java/io/ebeaninternal/api/LoadBeanRequest.java

```diff
@@ -19,30 +21,29 @@
   private final LoadBeanBuffer loadBuffer;
   private final String lazyLoadProperty;
   private final boolean loadCache;
-  private boolean loadedFromCache;
+  private final boolean alreadyLoaded;
```

Comment:
Introduce alreadyLoaded ... to indicate when lazy loading a bean that is already loaded that it should NOT use the L2 cache.